### PR TITLE
Update feedback.js

### PR DESCRIPTION
**Pull request by harshitjain0510**
On line 9 now, variable 'i' is starting from 3, so it's not working for more than 3 teachers. Change back 'i' to 1 and can add:-

if(x[i].getAttribute('name') == 'check'){continue;}

in loop before x[i].click()

It will work for any n number of teachers.

### CODE REVIEW

**Comment by grapheo12** [comment]
@harshitjain0510 Have you tested this yourself?
If yes I'll merge it right away.

File: feedback.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ try{
     }
 
     x = myframe.document.querySelectorAll('input[type="radio"]');
-    for (i = 3; i < x.length; i++){
+    for (i = 1; i < x.length; i++){
+        if(x[i].getAttribute('name') == 'check'){continue;}
```
Comment (grapheo12) [inline feedback.js:10]: Would you care to format this part? This is just my OCD :sweat:

**Review by grapheo12 — CHANGES_REQUESTED** [review]
Please do the needful

**Comment by harshitjain0510** [comment]
> @harshitjain0510 Have you tested this yourself?
> If yes I'll merge it right away.

Yes, tried and tested.

**Review by thealphadollar — APPROVED** [review]
LGTM. Tried and tested in times of need XD

**Pull request merged**